src/main/java/org/minions/devfund/katerinaanzoleaga/battleship/Ocean.java

```diff
@@ -0,0 +1,283 @@
+package org.minions.devfund.katerinaanzoleaga.battleship;
+
+import java.util.Formatter;
+import java.util.Random;
+
+/**
+ * Ocean class is the base of BattleShip game. It holds the ships to be sunk.
+ */
+public class Ocean {
+
+
+    private int shotsFired;
+    private int hitCount;
+    private int shipsSunk;
+    private static final int LENGTH = 20;
+    private static final int BATTLESHIPSNUMBER = 1;
+    private static final int BATTLECRUSIERSNUMBER = 1;
+    private static final int CRUSIERSNUMBER = 2;
+    private static final int LIGHTCRUISIERSNUMBER = 2;
+    private static final int DESTROYERSNUMBER = 3;
+    private static final int SUBMARINESNUMBER = 4;
+
+    private Ship[][] ships = new Ship[LENGTH][LENGTH];
+
+    /**
+     * Returns the Ships array.
+     * @return Array of ships
+     */
+    public Ship[][] getShipArray() {
+        return ships.clone();
+    }
+
+    /**
+     * Updates the Ships array.
+     * @param ships of the ocean.
+     */
+    public void setShips(final Ship[][] ships) {
+        this.ships = ships.clone();
+    }
+
+    /**
+     * Returns the number of shots the gamer fired.
+     * @return shots fired.
+     */
+    public int getShotsFired() {
+        return shotsFired;
+    }
+
+    /**
+     * Returns the number of hits.
+     * @return hit count
+     */
+    public int getHitCount() {
+        return hitCount;
+    }
+
+    /**
+     * Returns the number of Ships already sunk.
+     * @return shis sunk
+     */
+    public int getShipsSunk() {
+        return shipsSunk;
+    }
+
+    /**
+     * Ocean constructor.
+     */
+    public Ocean() {
+        shotsFired = 0;
+        hitCount = 0;
+        shipsSunk = 0;
+        this.fillOcean();
+    }
+
+    /**
+     * Fills the Ocean with empty ships.
+     */
+    private void fillOcean() {
+        for (int i = 0; i < this.ships.length; ++i) {
+            for (int j = 0; j < this.ships.length; ++j) {
+                EmptySea emptySea = new EmptySea();
+                emptySea.setBowRow(i);
+                emptySea.setBowColumn(j);
+                this.ships[i][j] = emptySea;
+            }
+        }
+    }
+
+    /**
+     * Prints the ocean with the values to display the game board and ships' status.
+     *
+     */
+    public void print() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 29 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)